boards/FSM/Inc/App/App_AcceleratorPedals.h

```diff
@@ -2,14 +2,10 @@
 
 #include <stdbool.h>
 
-// TODO: Experimentally determine the encoder values for the primary and
-// secondary APPS when the accelerator is fully pressed #666
-#define PAPPS_ENCODER_FULLY_PRESSED_VALUE (2500U)
-#define SAPPS_ENCODER_FULLY_PRESSED_VALUE (1800U)
-
-// If encoder exceeds the following values, we reset it back to zero (usually
-// due to underflowing)
-#define PAPPS_ENCODER_RESET_VALUE (3500U)
+#define PAPPS_ENCODER_FULLY_PRESSED_VALUE (1000U)
+#define PAPPS_ENCODER_UNPRESSED_VALUE (1353U)
```

Comment:
If a counter underflows it gets set to its max value. 
For a counter which counts up as the pedal is pressed (SAPPS), its possible we may underflow at 0% pedal pressed and get wrapped up to max value. I checked for this by checking if we've exceeded some value which is greater than  SAPPS_ENCODER_FULLY_PRESSED_VALUE, called SAPPS_ENCODER_RESET_VALUE. 
Since PAPPS counts downward from 1353 counter val at 0% pedal press to 1000 counter val at 100% pedal press it should never underflow (go below counter val of zero)
Tbh this is not the clearest way of implementing this